arangod/Aql/ExecutionBlockImpl.cpp

```diff
@@ -250,7 +252,9 @@ struct ExecuteSkipVariant<SkipVariants::DEFAULT> {
     // this function should never be executed
     TRI_ASSERT(false);
     // Make MSVC happy:
-    return {ExecutionState::DONE, {}, 0};
+    return std::make_tuple<ExecutionState, typename Executor::Stats, size_t>( // tupple, cannot use initializer list due to build failure
+      std::move(ExecutionState(ExecutionState::DONE)), {}, 0 // rvalue required for all elements to match constructor signature
```

Comment:
```suggestion
      ExecutionState(ExecutionState::DONE), {}, 0 // rvalue required for all elements to match constructor signature
```

Reply:
I would think so too, but MSVC was complaining with:
cannot convert argument 1 from 'arangodb::aql::ExecutionState' to 'arangodb::aql::ExecutionState &&'